ibm/resource_ibm_schematics_action.go

```diff
@@ -0,0 +1,1665 @@
+									"git_repo_url": &schema.Schema{
+										Type:         schema.TypeString,
+										Optional:     true,
+										Description:  "URL to the GIT Repo that can be used to clone the template.",
+										ValidateFunc: validation.IsURLWithHTTPorHTTPS,
+									},
+									"git_token": &schema.Schema{
+										Type:        schema.TypeString,
+										Optional:    true,
+										Description: "Personal Access Token to connect to Git URLs.",
+									},
+									"git_repo_folder": &schema.Schema{
+										Type:        schema.TypeString,
+										Optional:    true,
+										Description: "Name of the folder in the Git Repo, that contains the template.",
+									},
+									"git_release": &schema.Schema{
+										Type:        schema.TypeString,
+										Optional:    true,
+										Description: "Name of the release tag, used to fetch the Git Repo.",
+									},
+									"git_branch": &schema.Schema{
+										Type:        schema.TypeString,
+										Optional:    true,
+										Description: "Name of the branch, used to fetch the Git Repo.",
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+			"source_type": &schema.Schema{
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: InvokeValidator("ibm_schematics_action", "source_type"),
+				Description:  "Type of source for the Template.",
+			},
+			"command_parameter": &schema.Schema{
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "Schematics job command parameter (playbook-name, capsule-name or flow-name).",
+			},
+			"bastion": &schema.Schema{
+				Type:        schema.TypeList,
+				Optional:    true,
+				Description: "Complete target details with the user inputs and the system generated data.",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"name": &schema.Schema{
+							Type:        schema.TypeString,
+							Optional:    true,
+							Description: "Target name.",
+						},
+						"type": &schema.Schema{
+							Type:        schema.TypeString,
+							Optional:    true,
+							Description: "Target type (`cluster`, `vsi`, `icd`, `vpc`).",
+						},
+						"description": &schema.Schema{
+							Type:        schema.TypeString,
+							Optional:    true,
+							Description: "Target description.",
+						},
+						"resource_query": &schema.Schema{
+							Type:        schema.TypeString,
+							Optional:    true,
+							Description: "Resource selection query string.",
+						},
+						"credential_ref": &schema.Schema{
+							Type:        schema.TypeString,
+							Optional:    true,
+							Description: "Override credential for each resource.  Reference to credentials values, used by all the resources.",
+						},
+						"id": &schema.Schema{
+							Type:        schema.TypeString,
+							Optional:    true,
+							Computed:    true,
+							Description: "Target ID.",
+						},
+						"created_at": &schema.Schema{
```

Comment:
how come created_at ...created_by can become optional and computed

Reply:
Updated the attribute. This is due to the generated code. Will check if there is a known issue with the code generator or raise an issue